Assets/Scripts/Enemy/EnemySpawn.cs

```diff
@@ -77,22 +92,23 @@ void StopSpawningWave()
     {
         canSpawn = false;
         startSpawn = false;
-        enemiesSpawned = 0;
+        
     }
     // Update is called once per frame
     void Update()
     {
-        if (startSpawn)
+        if (startSpawn && !_timeout)
         {
             SpawnWave();
         }
         else
         {
             currentNumEnemies = GameObject.FindGameObjectsWithTag("Enemy").Length;
-            UIManager.enemiesLeft = currentNumEnemies;
+            UIManager.enemiesLeft = enemiesPerWave - (enemiesSpawned - currentNumEnemies); ;
             //Debug.Log(currentNumEnemies);
+            //Debug.Log(enemiesSpawned);
```

Comment:
are these debugs convenient enough to leave in

Reply:
mb should delete